### ISSUE TITLE
Can haxx the "Second Select" Activity

### ISSUE THREAD

**Issue by aburd** [scootyboots/css-selector-training-platform#4]
## Description

Can finish the lesson in an unconventional way

1. Select the second blog post using the way you're not supposed to ".Blog-post:nth-child(2)"
![Screenshot from 2022-03-31 23-55-26](https://user-images.githubusercontent.com/6701630/161085994-2c760167-ba5d-4344-9e29-1a77be440c3e.png)
2. Use the appended selector to complete the lesson: ".Blog-post.--selected-from-answer"
![Screenshot from 2022-03-31 23-55-49](https://user-images.githubusercontent.com/6701630/161086018-473618eb-9acc-4056-85e6-9ad3c7e114d5.png)

**Comment by scootyboots** [comment]
Oooh that's a good point, most any exercise could be exploited in this way. 

I decided to use a custom attribute `data-selected-from-answer` instead of a class so hopefully, it won't be as obvious to people that the answer checks could be exploited in this way. In general it seems like a bad idea to have classes being augmented on anything inside the browser area, unless my the training site itself. 

I've also added a check to see if the CSS selector answer utilizes the new custom attribute and bail out if so - NO CHEATERS ALLOWED 7338a813448496b20468591f73af3d9846805ca5
